sale_coupon_criteria_multi_product/models/sale_coupon_criteria.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class SaleCouponCriteria(models.Model):
+    _name = "sale.coupon.criteria"
+    _description = "Coupon Multi Product Criteria"
+
+    program_id = fields.Many2one(comodel_name="sale.coupon.program",)
+    rule_min_quantity = fields.Integer(
+        string="Minimum Quantity",
```

Comment:
For being friendly with tree header:
```suggestion
        string="Min. Qty.",
```